r-package/grf/R/average_partial_effect.R

```diff
@@ -13,6 +18,7 @@
 #'               estimate the ATE. WARNING: For valid statistical performance,
 #'               the subset should be defined only using features Xi, not using
 #'               the treatment Wi or the outcome Yi.
+#' @param num.trees.for.variance Number of trees used to estimate Var[Wi | Xi = x].
```

Comment:
It's generally good to make unrelated changes like this in a separate PR. At the least, maybe we could add a note to the PR description noting these additional changes?